CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.17.1 (2021-05-05)
```

Comment:
```suggestion
# 4.17.1 (2021-05-06)
```
Updating the date since I assume this will go out tomorrow. If you catch this tonight and want to get this out, I'd disregard both suggestions.